website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -196,8 +196,8 @@ These ordering rules are subject to change.
 | 1.1.0 | Supports [Timeseries & Aggregations](#timeseries-and-aggregations). Added support for type `Int8` for `id`. |
 | 1.0.0 | Supports [`indexerHints`](/developing/creating-a-subgraph/#indexer-hints) feature to prune subgraphs |
 | 0.0.9 | Supports `endBlock` feature |
-| 0.0.8 | Added support for polling [Block Handlers](developing/creating-a-subgraph/#polling-filter) and [Initialisation Handlers](developing/creating-a-subgraph/#once-filter). |
-| 0.0.7 | Added support for [File Data Sources](developing/creating-a-subgraph/#file-data-sources). |
+| 0.0.8 | Added support for polling [Block Handlers](creating-a-subgraph/#polling-filter) and [Initialisation Handlers](creating-a-subgraph/#once-filter). |
+| 0.0.7 | Added support for [File Data Sources](creating-a-subgraph/#file-data-sources). |
```

Comment:
```suggestion
| 0.0.8 | Added support for polling [Block Handlers](/developing/creating-a-subgraph/#polling-filter) and [Initialisation Handlers](/developing/creating-a-subgraph/#once-filter). |
| 0.0.7 | Added support for [File Data Sources](/developing/creating-a-subgraph/#file-data-sources). |
```
I don't think we should use relative URLs. They break when we move pages around.